tests/unit/plugins/europeana/record.spec.js

```diff
@@ -0,0 +1,75 @@
+import nock from 'nock';
+import getRecord from '../../../../plugins/europeana/record';
+
+const axios = require('axios');
+axios.defaults.adapter = require('axios/lib/adapters/http');
+
+const apiUrl = 'https://api.europeana.eu';
+const apiEndpoint = '/api/v2/record/test/thisisatest.json';
+const apiKey = 'abcdef';
+const path = 'test/thisisatest';
+
+describe('plugins/europeana/record', () => {
+  describe('getRecord()', () => {
+
+    describe('when trying to request an unknown record', () => {
+      const errorMessage = 'Invalid record identifier';
+      beforeEach('stub API response', () => {
+        nock(apiUrl)
+          .get(apiEndpoint)
+          .query(true)
+          .reply(400, {
+            record: null,
```

Comment:
Omit `record` as the API does not return it, but add `"success": false` which it does return.